tests/assets/operators/honeybee-radiance/operator.yaml

```diff
@@ -1,4 +1,4 @@
-name: honeybee-radiance
+# name: honeybee-radiance
```

Comment:
Is this mean that operator doesn't need a name?

Reply:
Nope. This means I need to check my typos :upside_down_face: It raises an error when you try to read it.